readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
```

Comment:
**Suggestion (blocking, clarity):** Maybe mention that underneath the project name they can include the project website and project owner name. The project name itself isn't optional. 😄